app/src/auth/components/TextInput/index.jsx

```diff
@@ -7,23 +7,48 @@ import TextField from '../TextField'
 
 type Props = FormControlProps & {
     className?: ?string,
+    onBlur?: (SyntheticInputEvent<EventTarget>) => void,
```

Comment:
Copied from `$shared/components/TextInput`. Otherwise this component is the same but the nested `TextField` comes inside the `$auth` folder.